### DISCUSSION TITLE
that RSS does not refresh

### DISCUSSION THREAD

**Discussion by ludo453** [FreshRSS/FreshRSS#3813] [General]
Hello,
I am under version 1.19.0-dev and I have the impression that RSS does not refresh.

**Comment by Frenzie** [comment]
See https://freshrss.github.io/FreshRSS/en/admins/08_FeedUpdates.html

**Reply by ludo453** [reply]
Thanks but now you need a script for that?

**Reply by Frenzie** [reply]
When you say "now", do you mean there's something you used to do that isn't working anymore?

**Reply by ludo453** [reply]
no I had another version and I didn't do anything before it did it all by itself and when I reinstalled I got the dev version

**Reply by ludo453** [reply]
hi it is the edge version and not the master which was set by default in github.

**Reply by Frenzie** [reply]
Glad you figured it out.

**Reply by ludo453** [reply]
ok but why not fall on the master as before?

**Reply by Frenzie** [reply]
I think that should be taken care of automatically? Pinging @Alkarex https://github.com/FreshRSS/FreshRSS/pull/3589

**Reply by Alkarex** [reply]
We now have a single branch *edge* (previously *dev*), which is our continuous release. We make regular stable releases (also tagged as *latest* in git) (previously *master*).
We are not using a *master* branch anymore.

**Reply by Alkarex** [reply]
@Frenzie I have not understood the question / problem.

**Reply by ludo453** [reply]
the problem is before with the master version it was updating the RSS without problem and now it does but for all my RSS with the dev version.

**Reply by Alkarex** [reply]
Do I understand correctly that you now are running an *edge* version that is working fine? All is good then :-)

**Reply by ludo453** [reply]
no precisely all the RSS does not refresh in master version it was good but not now

**Reply by Alkarex** [reply]
How did you deploy FreshRSS, with which database, and on which platform? Is the manual refresh working? How is your refresh cron set up?

**Reply by ludo453** [reply]
in manual yes no cron and on mariadb.

**Reply by Alkarex** [reply]
If you do not have a cron then feeds cannot be automatically refreshed. Cf. link that Frenzie sent you.

**Reply by ludo453** [reply]
yes i saw but pk now need cron?

**Reply by Frenzie** [reply]
What specifically is it that you used to do that is no longer working?

**Reply by ludo453** [reply]
before I did nothing it refreshes itself and sometimes it refreshes and other force to do it manually

**Reply by Alkarex** [reply]
Some feeds have a WebSub mechanism pushing updates instantly (for instance http://rss.slashdot.org/slashdot/slashdotMain ). But for most feeds, a cron job is required to allow automatic retrieval of new articles. This has always been the case.

**Reply by ludo453** [reply]
I never noticed that

**Comment by Frenzie** [comment]
I didn't either since I thought it was about cron.
